ajax/libs/react-extras/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "react-extras",
+  "title": "react-extras",
+  "description": "Useful components and utilities for working with React",
+  "author": {
+    "name": "Sindre Sorhus",
+    "url": "sindresorhus.com"
+  },
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/sindresorhus/react-extras"
+  },
+  "npmName": "react-extras",
+  "npmFileMap": [
+    {
+      "basePath": "source",
```

Comment:
`source` directory does not exist in the published package. Did you want to use `dist`?

```
ls node_modules/react-extras/dist
auto-bind.js     body-class.js    choose.js        class-names.js   element-class.js for.js           if.js            image.js         index.js         root-class.js
```